packages/core/src/internal/io.js

```diff
@@ -161,7 +161,7 @@ export function select(selector = identity, ...args) {
   channel(pattern, [buffer])    => creates a proxy channel for store actions
 **/
 export function actionChannel(pattern, buffer) {
-  if (process.env.NODE_ENV === 'development') {
+  if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
```

Comment:
actually I've wanted to migrate every `process.env.NODE_ENV === 'development'` check to `process.env.NODE_ENV !== 'production'`, would u like to do this as part of this PR?

Reply:
Sure, I'd like to.